WickedEngine/shaders/ShaderInterop_EmittedParticle.h

```diff
@@ -73,6 +73,7 @@ CBUFFER(EmittedParticleCB, CBSLOT_OTHER_EMITTEDPARTICLE)
 
 	float		xSPH_poly6_constant;	// precomputed Poly6 kernel constant term
 	float		xSPH_spiky_constant;	// precomputed Spiky kernel function constant term
+	float		xSPH_visc_constant;	    // precomputed viscosity kernel function constant term
```

Comment:
This could be a problem putting this value here. The members of constant buffer structures must be following the constant buffer packing rules, that's why the members are grouped into 16-byte regions (for example 4 floats following each other). The constant buffers must be padded explicitly, some info is in the documentation: https://github.com/turanszkij/WickedEngine/blob/master/Content/Documentation/WickedEngine-Documentation.md#shaders

Reply:
It should be packed correctly now.